packages/preview-middleware-client/src/adp/init-dialogs.ts

```diff
@@ -59,12 +62,42 @@ export const initDialogs = (rta: RuntimeAuthoring, syncViewsIds: string[]): void
  * @param overlays Control overlays
  * @param syncViewsIds Runtime Authoring
  *
- * @returns boolean
+ * @returns boolean whether menu item is enabled or not
  */
 export const isControllerExtensionEnabled = (overlays: ElementOverlay[], syncViewsIds: string[]): boolean => {
+    if (overlays.length === 0 || overlays.length > 1) return false;
+
     const clickedControlId = FlUtils.getViewForControl(overlays[0].getElement()).getId();
+    const isClickedControlReuseComponent = isReuseComponent(clickedControlId);
 
-    return overlays.length <= 1 && !syncViewsIds.includes(clickedControlId);
+    return !syncViewsIds.includes(clickedControlId) && !isClickedControlReuseComponent;
+};
+
+/**
+ * Function that checks if clicked control is from view which is reuse component
+ *
+ * @param clickedControlId id of the clicked control
+ * @returns boolean if clicked control is from reused component view
+ */
+export const isReuseComponent = (clickedControlId: string): boolean => {
+    const version = sap.ui.version;
```

Comment:
Global usage should be avoided. Please use `sap/ui/VersionInfo` instead.

Reply:
fixed in #[9aa908c](https://github.com/SAP/open-ux-tools/pull/1941/commits/9aa908cf6e0415b18d849970bd855e53f98ef3db)